Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocListRenderer.cs

```diff
@@ -121,18 +121,14 @@ public override sealed string CssClassReadOnly
     {
       ArgumentUtility.CheckNotNull("htmlHeadAppender", htmlHeadAppender);
 
-      htmlHeadAppender.RegisterUtilitiesJavaScriptInclude();
-
+      htmlHeadAppender.RegisterWebClientScript();
+      htmlHeadAppender.RegisterObjectBindingWebClientScript();
```

Comment:
I wonder if it would be better to have `RegisterObjectBindingWebClientScript` also call `RegisterWebClientScript` since we need the utilities all the time anyway?